Library/Homebrew/rubocops/extend/formula.rb

```diff
@@ -472,6 +473,32 @@ def formula_tap
         match_obj[1]
       end
 
+      # Returns whether the given formula exists in the given style exception list.
+      # Defaults to the current formula being checked.
+      def tap_style_exception?(list, formula = nil)
+        if @tap_style_exceptions.blank? && formula_tap.present?
```

Comment:
Probably can't rely on ActiveSupport i.e. blank/present here 😭

Reply:
Oh yeah, probably not. `.nil?` works, though, right?